deno/rest/v10/channel.ts

```diff
@@ -585,7 +585,12 @@ export type RESTPostAPIChannelMessagesThreadsJSONBody = AddUndefinedToPossiblyUn
  * https://discord.com/developers/docs/resources/channel#start-thread-in-forum-channel
  */
 export type RESTPostAPIGuildForumThreadsJSONBody = RESTPostAPIChannelMessagesThreadsJSONBody &
-	RESTPostAPIChannelMessageJSONBody;
+	RESTPostAPIChannelMessageJSONBody & {
```

Comment:
`RESTPostAPIChannelMessageJSONBody` is no longer needed. in the merged type. Also, for the JSON body, the type for message should be `RESTPostAPIChannelMessageJSONBody`